lib/cap_profile_id_rewriter.rb

```diff
@@ -1,102 +0,0 @@
-# TODO: This code could be migrated into a runner script in `scripts/`
```

Comment:
This file is required whenever a prod-db dump is loaded into a -dev or -qa system.  It is used to sync up the CAP-API author data.  This can also happen when the CAP team sync data across their -dev or -qa systems.  Removing this file will not be approved in this PR.  For additional review/acceptance of other changes in this PR, drop this commit.  If there is an argument to be had about removing this, move this commit to another PR.  AFAICT, this commit is an independent issue that has no clear connection to the other commits in this PR.